media/js/src/homebanner.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+export const HomeBanner: React.FC = () => {
+
+    return (<>
+        <header className="main-banner banner-hero">
```

Comment:
add `data-testid={'HomeBanner'}` to the header tag and this will make it pass the test.